Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs

```diff
@@ -116,6 +116,36 @@ public bool RenderMotionControllers
             private set { renderMotionControllers = value; }
         }
 
+        [SerializeField]
+        [Tooltip("Default Left Controller Model")]
+        private GameObject leftControllerModel;
+
+        /// <summary>
+        /// Enable and configure the controller rendering for the Mixed Reality Toolkit
+        /// </summary>
+        public GameObject LeftControllerModel
```

Comment:
I think the controller models should be moved into the controller configuration profiles.